### ISSUE TITLE
.dev gTLD long term stability concerns

### ISSUE THREAD

**Issue by picnoir** [nix-rfc-canonical-domain/rfcs#10]
I'm concerned about the `.dev` gTLD long-term stability. I feel like we're trading domain name briefty against stability with the current proposal.

Here are my main concerns with the .dev gTLD:

- This domain is currently owned by Charleston Road Registry, a Google subsidiary. At the moment, there's a strong dependency between `.dev` and a massive tech company whose domain name selling side quest is clearly not their main business. `.org` is currently owned by [PIR](https://en.wikipedia.org/wiki/Public_Interest_Registry) (but already changed hands in the past), a business focused on domain name registrations.
- It's a pretty recent gTLD, it's only been available for the general public for 5 years up until now. `.org` has been around for 39 years.
- `.dev` is way more niche than .org in terms of the number of registrations. There seem to be [~400k registered .dev domains](https://ntldstats.com/tld/dev) at the moment. Compare that with the [~10M .org registrations in 2012](https://en.wikipedia.org/wiki/.org#Registrations) (I can't find any recent numbers for .org :disappointed:  ) 

All in all, I'm pretty confident .org will be around 20 years from now. I can't say the same for .dev.

Domain migration is painful on the web. If we decide to migrate somewhere else, I think we should be careful to pick a TLD we're sure won't disappear in the long term. nixos.org has been registered for 17 years, I think it's fair to assume we're operating on a long-term timescale here.

**Comment by byrongibson** [comment]
> nixos.org has been registered for 17 years

I also tend to favor NixOS.org for that reason too.  Massive accumulated SEO value, evidence of [Lindy Effect](https://en.wikipedia.org/wiki/Lindy_effect), plus general name recognition associating both Nix and NixOS with that domain.  I think those factors are far more valuable to the project than anything moving to nix.dev could provide.

To solve the nix vs nixos issue, it would probably be possible to rework the content on nixos.org to more clearly differentiate between Nix and NixOS.  Currently the site information architecture commingles the two projects in ways that may cause newcomers to also mentally commingle them, or force folks who only want docs on one project to wade through pages on the other.  But that problem should be solvable with a site information architecture rework - create more distinct site sections, subdomains, and/or url tree hierarchies of the website for each project.  

That may be more work up front than just redirecting to a new domain, but I think it preserves far more value for the project than it costs.

**Comment by fricklerhandwerk** [comment]
@byrongibson work on information architecture is slowly progressing: https://discourse.nixos.org/t/zurich-23-11-zhf-hackathon-and-ux-workshop-report/37848

**Comment by byrongibson** [comment]
> @byrongibson work on information architecture is slowly progressing: https://discourse.nixos.org/t/zurich-23-11-zhf-hackathon-and-ux-workshop-report/37848

Oh thanks, excellent work and writeup, glad to see it.  Yeah I hope this problem can be solved with a UI/UX rework than by abandoning an almost 2-decade domain.

Also one other suggestion, maybe officially rename the overall project to Nix/OS, making it more clear to newcomers there are multiple components and not just an OS.  Then continue using "Nix" to refer to the package manager, "Nix Language" or "nix-lang" for the config language, and NixOS for the OS.  Put Nix/OS in big letters in site's Hero Section at the top to make it clear that "nixos.org" means Nix/OS, inclusive

Naming things is hard, but can communicate quite a lot in minimal bits.

**Comment by infinisil** [comment]
Yeah agreed, I'm thinking of just closing this RFC draft, deem it not worth. Instead we can achieve effectively the same result by
- Moving nix.dev to docs.nixos.org: https://github.com/NixOS/nix.dev/issues/869
- Improving nixos.org to be about the entire Nix platform, not just focus on Nix
